scripts/deploy.sh

```diff
@@ -127,6 +128,12 @@ if [[ "${DEPLOY_OA}" == "yes" ]]; then
     run_ansible haproxy-install.yml
   fi
 
+  # Apply security hardening
+  if [[ "$DEPLOY_HARDENING" == "yes" ]]
+  then
```

Comment:
if [[ "$DEPLOY_HARDENING" == "yes" ]]
then
 set it to true
else
 set it to false
fi